GraalVM/CE/1.0.0-rc7/README.md

```diff
@@ -0,0 +1,63 @@
+
+# Supported tags and respective Dockerfile links
+* 1.0.0-rc7 (1.0.0-rc7/Dockerfile)
```

Comment:
I don't think this needs to be documented here as you're already in the `1.0.0-rc7` directory.

Reply:
The idea is to add links to other tags, like on https://hub.docker.com/_/openjdk/.

But as this is the first tag, we can remove it for now.

Reply:
Yeah, once you have multiple tags then this makes sense.